vault/identity_store_oidc_provider_test.go

```diff
@@ -16,12 +16,50 @@ import (
 )
 
 func TestOIDC_Path_OIDC_Authorize(t *testing.T) {
+	c, _, _ := TestCoreUnsealed(t)
+	ctx := namespace.RootContext(nil)
+	storage := new(logical.InmemStorage)
+
+	// Create a key
+	resp, err := c.identityStore.HandleRequest(ctx, &logical.Request{
+		Path:      "oidc/key/test-key",
+		Operation: logical.CreateOperation,
+		Data:      map[string]interface{}{},
+		Storage:   storage,
+	})
+	expectSuccess(t, resp, err)
+
+	// Create an entity
+	resp, err = c.identityStore.HandleRequest(ctx, &logical.Request{
+		Path:      "entity",
+		Operation: logical.UpdateOperation,
+		Data: map[string]interface{}{
+			"name": "test-entity",
+		},
+	})
+	expectSuccess(t, resp, err)
+	assert.NotNil(t, resp.Data["id"])
+	entityID := resp.Data["id"].(string)
+
+	// Create a group
+	resp, err = c.identityStore.HandleRequest(ctx, &logical.Request{
+		Path:      "group",
+		Operation: logical.UpdateOperation,
```

Comment:
nit: since we are doing a create maybe this should be `CreateOperation`?

Reply:
See comment above.